couchbase/json-buckets.yml

```diff
@@ -0,0 +1,968 @@
+
+      - name: curr.connections
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.curr_connections[0]
+        type: long_gauge
+        label: current connections
+        description: Current bucket connections
+
+      - name: curr.items.tot
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.curr_items_tot[0]
+        type: long_gauge
+        label: items
+        description: Total number of items
+
+      - name: curr.items
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.curr_items[0]
+        type: long_gauge
+        label: acive items
+        description: Number of active items in memory
+
+      - name: decr.hits
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.decr_hits[0]
+        type: long_gauge
+        label: decrement hits
+        description: Decrement hits
+
+      - name: decr.misses
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.decr_misses[0]
+        type: long_gauge
+        label: decrement misses
+        description: Decrement misses
+
+      - name: delete.hits
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.delete_hits[0]
+        type: long_gauge
+        label: delete hits
+        description: Delete hits
+
+      - name: delete.misses
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.delete_misses[0]
+        type: long_gauge
+        label: delete misses
+        description: Delete misses
+
+      - name: disk.commit.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.disk_commit_count[0]
+        type: long_gauge
+        label: disk commits
+        description: Disk commits
+
+      - name: disk.update.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.disk_update_count[0]
+        type: long_gauge
+        label: disk updates
+        description: Disk updates
+
+      - name: disk.write.queue
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.disk_write_queue[0]
+        type: long_gauge
+        label: disk writes
+        description: Disk write queue depth
+
+      - name: ep.bg.fetched
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_bg_fetched[0]
+        type: long_gauge
+        label: disk reads
+        description: Disk reads per second
+
+      - name: ep.cache.miss.rate
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_cache_miss_rate[0]
+        type: long_gauge
+        label: cache miss rate
+        description: Cache miss rate
+
+      - name: ep.cache.miss.ratio
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_cache_miss_ratio[0]
+        type: long_gauge
+        label: cach miss ratio
+        unit: "%"
+        description: Cache miss ratio
+
+      - name: ep.dcp.fts.backoff
```

Comment:
Cryptic again. And in few more places below.

Reply:
The full description is "Number of backoffs for fts DCP connections"
The abbr "fts", "DCP" is used also in full description, so i will leave those abbr in our metrics name.